### PR TITLE
enables S3v4 support

### PR DESCRIPTION
**Pull request by eLRuLL**
**TL;DR -** [ A short summary of what this PR does and why ]

JIRA: [JIRA-XXXX](https://openedx.atlassian.net/browse/JIRA-XXXX)

**What changed?**

- [ More in depth breakdown of changes ]
- [ Peripheral things that got changed ]
- [ etc... ]

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @eLRuLL! I've created [OSPR-5667](https://openedx.atlassian.net/browse/OSPR-5667) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

**Comment by natabene** [comment]
@eLRuLL Thank you for your contribution. Please let me know once it is ready for our review.

**Comment by eLRuLL** [comment]
@natabene ty for the reminder, this was a draft but I don't think something needed at this point, I'll be closing this 👍

**Comment by edx-webhook** [comment]
@eLRuLL Even though your pull request wasn’t merged, please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-ora2&entry.867055334=https://github.com/edx/edx-ora2/pull/1606&entry.1484655318=https://github.com/eLRuLL&entry.752974735=2021-03-12+00:53&entry.1917517419=2021-03-16+17:37&entry.2133058324=No) so we can improve your experience in the future.
